### PR TITLE
Heavy Crates + Updated crate designs

### PR DESCRIPTION
**Pull request by Rusty-Box**
**Heavy crates** are stronger, metal reinforced crate variants which can only be destroyed through bomb explosions, larger Crusher variants (e.g. Krosh) and Earth Tux's rock ability.

Aside heavy crates, this PR updates the look for each crate type by adding outlines and other minor tweaks.

### CODE REVIEW

File: data/images/tiles.strf [hunk 1]
```diff
@@ -1620,6 +1620,15 @@
...
     (object-data "(breakable #t) (sprite \"images/objects/bonus_block/brickWeb.sprite\")")
   )
+  (tile
+    (id 4434)
+    (images
+      "tiles/blocks/brick3.png"
+    )
+    (solid #t)
+    (object-name "brick")
+    (object-data "(breakable #t) (sprite \"images/objects/bonus_block/heavy-brick.sprite\")")
```
Comment (Semphriss) [inline data/images/tiles.strf:1630]: `heavy-brick` and the sprite doesn't need to be specified

**Review by Semphriss — CHANGES_REQUESTED** [review]
Two tiny changes :^)

File: data/images/objects/bonus_block/heavy-brick.sprite [hunk 2]
```diff
@@ -0,0 +1,42 @@
+(supertux-sprite
+      (action
```
Comment (tobbi) [inline data/images/objects/bonus_block/heavy-brick.sprite:2]: Make indentation consistent please.

Comment (Rusty-Box) [reply data/images/objects/bonus_block/heavy-brick.sprite]: Should be fixed now with the latest commit.

**Review by Semphriss — APPROVED** [review]

**Comment by bruhmoent** [comment]
Crates that can be destroyed only by bombs... interesting

**Comment by Semphriss** [comment]
Bombs, Rock Tux in his rocky mode (useful with #1820) and big icecrushers can destroy heavy bricks.

**Pull request merged**